src/Document/TextElement.ts

```diff
@@ -337,53 +382,57 @@ export default class TextElement extends RenderedElement {
 		const yAttr = child.getAttribute('y');
 		const dxAttr = child.getAttribute('dx');
 		const dyAttr = child.getAttribute('dy');
-		const textAnchor = child.getAttribute('text-anchor').getString('start');
+		const customFont = child.getStyle('font-family').getDefinition<FontElement>();
+		const isRTL = Boolean(customFont) && customFont.isRTL;
 
-		if (i === 0 && child.type !== 'textNode') {
+		if (i === 0) {
+			// First children inherit attributes from parent(s). Positional attributes
+			// are only inherited from a parent to it's first child.
 			if (!xAttr.hasValue()) {
-				xAttr.setValue(textParent.getAttribute('x').getValue('0'));
+				xAttr.setValue(TextElement.inheritAttribute(child, 'x'));
```

Comment:
Is child can be not `TextElement`? Is `getInheritAttrubute` must be a static method?

Reply:
I think at this place `child` is always a `TextElement` because of the
```
if (typeof child.measureText !== 'function') {
  return child;
}
```
check before. I've converted it to an instance method now.